### PR TITLE
Add Fairing release team 

### PR DESCRIPTION
**Pull request by jinchihe**
Add Fairing release team

### CODE REVIEW

**Comment by jinchihe** [comment]
/assign @jlewi 

Thanks!

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -860,6 +861,7 @@ orgs:
             - richardsliu
             members:
             - Jeffwan
+            - jinchihe
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml:864]: Please create a fairing release-team which has access to just the repos needed for fairing.

Comment (jlewi) [reply github-orgs/kubeflow/org.yaml:864]: Please remove yourself from this list as we are moving away from giving folks broad access and instead giving more narrowly scoped access.

Comment (jinchihe) [reply github-orgs/kubeflow/org.yaml:864]: OK

**Comment by jinchihe** [comment]
@jlewi Updated as your comments. Thanks.

**Pull request merged**